ByteUtil.java

```diff
@@ -0,0 +1,25 @@
+
+import java.nio.charset.StandardCharsets;   //Importing UTF-8 Character Encoding
+
+public class ByteUtil {
+    public String convert(byte[] data){ // Method to convert byte[] array to string
```

Comment:
Make this a static method so you don't have to create an object to use it. Ideally, you would be able to use it like this:
`String s = ByteUtil.convert(data);`